draft-ietf-rtcweb-jsep.xml

```diff
@@ -1964,22 +1963,87 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
           prohibited in the creation of answers.</t>
         </section>
 
-        <section title="Subsequent Answers" anchor="sec.subsequent-answers"></section>
+        <section title="Subsequent Answers" anchor="sec.subsequent-answers">
+          <t>When createAnswer is called a second (or later) time, or is called
+          after a local description has already been installed, the processing
+          is somewhat different than for an initial answer.</t>
+
+          <t>If the initial answer was not applied using setLocalDescription,
+          meaning the PeerConnection is still in the "have-remote-offer" state,
+          the steps for generating an initial answer should be followed, subject
+          to the following restriction: <list style="symbols">
+            <t>The fields of the "o=" line MUST stay the same except for the
+            &lt;session-version&gt; field, which MUST increment if the session
+            description changes in any way, including the addition of
+            ICE candidates.</t>
```

Comment:
I'm trying to figure out how candidates could have been added. Doesn't gathering start with sLD?